pkg/intcode/intcode.go

```diff
@@ -130,11 +106,15 @@ func (p *program) extractArguments() (op opcode, arg1, arg2, arg3 int) {
 }
 
 func (p *program) result() int {
+	res := p.instructions[0]
+
 	if len(p.res) > 0 {
-		return p.res[len(p.res)-1]
+		log.Println("from res")
+		res = p.res[len(p.res)-1]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '1', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)